tools/cold-backup/README.md

```diff
@@ -49,15 +55,15 @@ kubectl run producer-perf -it --image="$CLIENT_IMAGE" --rm="true" --restart="Nev
     --throughput -1 --producer-props acks=1 bootstrap.servers="my-cluster-kafka-bootstrap:9092"
 
 # run backup procedure
-./tools/cold-backup/run.sh backup -n myproject -c my-cluster -t /tmp/my-cluster.zip
+./tools/cold-backup/run.sh backup -n myproject -c my-cluster -t /tmp/my-cluster.tgz
```

Comment:
by changing to tgz I would say that requirements in the README could be changed by removing the zip and unzip as needed?

Reply:
Right, thanks for spotting this.